pkg/chain/ethereum/connect.go

```diff
@@ -162,3 +174,20 @@ func addClientWrappers(
 
 	return loggingClient
 }
+
+// FIXME Rip this back out to connector_ethereum.go after tBTC handle refactor.
+func (ec *ethereumChain) buildTBTC(tbtcSystemAddress string) chain.TBTCHandle {
+	if len(tbtcSystemAddress) > 0 {
+		tbtcEthereumChain, err := WithTBTCExtension(
+			ec,
+			tbtcSystemAddress,
+		)
+		if err != nil {
+			return nil
+		}
+
+		return tbtcEthereumChain
+	}
+
+	return nil
+}
```

Comment:
It is unclear to me if we can move this piece to the connector. The connector will have a reference to `chain.Handle` and `chain.Handle` will give access to `BondedECDSAKeepManager` with `TBTCApplicationHandle()` but the code will remain more or less the same here. We will rename the function and remove the parameter but... that's all?

Reply:
Sort of, yeah, because the application handle will wrap the contract instance. The only reason we do this contortion right now is because the tbtc handle is a distinct object that also requires an external instantiation of the contract instance. We make that internal with the TBTCApplicationHandle refactor, so the caller can just fetch it from chain.Handle instead of needing a triple return from ethereum.Connect to get to it.